Source/WebCore/platform/graphics/filters/FilterImageTargetSwitcher.cpp

```diff
@@ -65,20 +65,26 @@ void FilterImageTargetSwitcher::beginClipAndDrawSourceImage(GraphicsContext& des
     }
 }
 
-void FilterImageTargetSwitcher::endClipAndDrawSourceImage(GraphicsContext& destinationContext)
+void FilterImageTargetSwitcher::endClipAndDrawSourceImage(GraphicsContext& destinationContext, const DestinationColorSpace& colorSpace)
 {
     if (auto* context = drawingContext(destinationContext))
         context->restore();
 
-    endDrawSourceImage(destinationContext);
+    endDrawSourceImage(destinationContext, colorSpace);
 }
 
-void FilterImageTargetSwitcher::endDrawSourceImage(GraphicsContext& destinationContext)
+void FilterImageTargetSwitcher::endDrawSourceImage(GraphicsContext& destinationContext, const DestinationColorSpace& colorSpace)
 {
     if (!m_filter)
         return;
 
     FilterResults results;
+#if USE(CAIRO)
```

Comment:
I would appreciate a comment here to why this is different in CG and Cairo. (Skia will be more like CG here)